app/src/main/kotlin/com/simplemobiletools/musicplayer/extensions/Context.kt

```diff
@@ -185,6 +186,10 @@ fun Context.getAlbumTracksSync(albumId: Long): ArrayList<Track> {
         projection.add(Audio.Media.BUCKET_DISPLAY_NAME)
     }
 
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
use helper functions like isRPlus instead of these please

Reply:
Done.